csrankings-q.csv

```diff
@@ -109,6 +109,7 @@ Qiong Luo 0001,HKUST,https://www.cse.ust.hk/~luo,SqAsppUAAAAJ
 Qirong Ho,MBZUAI,https://mbzuai.ac.ae/study/faculty/qirong-ho,tR3AZbwAAAAJ
 Qirun Zhang,Georgia Institute of Technology,http://helloqirun.github.io,u0TKjhIAAAAJ
 Qishi Wu,NJIT,http://cs.njit.edu/people/wu.php,C__DNg8AAAAJ
+Qiufu Li,Shenzhen University,https://github.com/LiQiufu,kpoePUQAAAAJ

```

Comment:
This is not a suitable home page. Please replace with a personal page or an institutional page.